package.json

```diff
@@ -4,16 +4,15 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "npm run remove-dist && npm run build:production && npm run build:static",
+    "build": "npm run build:production && npm run build:static",
     "build:production": "webpack -p --mode=production",
     "build:test": "webpack -p --mode=production --include-tests",
-    "build:dev": "npm run remove-dist && npm run build:static && webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo",
+    "build:dev": "webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo && npm run build:static",
```

Comment:
This is not specific to this issue, but would be great to cleanup here I think:
Let's use the `-d` and `-p` options and remove the verbose flags they resolve to. 

See https://webpack.js.org/api/cli/#shortcuts

Reply:
I much prefer the explicit flags—we never have to manually type these, so it's not saving us any time when typing. But it's easier to read and reason about, especially if you are new to the project or unfamiliar with `webpack-cli`'s API. I'd rather we kept the long ones 😄

Reply:
Hmm, that's fine with me I guess. In that case, we should remove the `-p` we're using since that's redundant to use with the flags.